package.json

```diff
@@ -145,9 +145,6 @@
     "typescript": "^4.9.4",
     "webpack": "^5.75.0"
   },
-  "peerDependencies": {
-    "ember-source": "^3.28.0 || ^4.0.0"
-  },
```

Comment:
These lines were added when I ran `ember-cli-update` to update `ember-cli` and `ember-source` to `v4.9`.

I wasn't sure if the lines mean a breaking change for consuming apps below Ember 3.28, so I removed them from this pull request. (Consider adding them back after `ember-container-query@v3.0.0` is released.)

More information at https://github.com/ember-cli/ember-cli/pull/10024.